TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -281,20 +339,23 @@ public object GetValue(object o, SymbolTable table)
                 foreach (var data in setStatement.StorageAreaWrites)
                 {
                     variablesWrittenRaw.Add(
-                        node.SymbolTable.DataEntries.Where(x => x.Key == data.MainSymbolReference.Name).SelectMany(x => x.Value).ToList());
+                        node.SymbolTable.GetVariablesExplicitWithQualifiedName(data.MainSymbolReference.URI).Select(x => x.Value).ToList());
```

Comment:
See with what Cosmin did, we now shloud be able to get this information without calling SymbolTable.